.pre-commit-config.yaml

```diff
@@ -99,6 +99,24 @@ repos:
       - id: markdownlint
         args: ["--fix"]
 
+  # Run tools on Jupyter notebooks
+
+  # strip output from Jupyter notebooks
+  - repo: https://github.com/kynan/nbstripout
+    rev: 0.5.0
+    hooks:
+      - id: nbstripout
+
+  # py-like tools
+  - repo: https://github.com/nbQA-dev/nbQA
+    rev: 1.3.1
+    hooks:
+      - id: nbqa-pyupgrade
+        args: [--py39-plus]
+      - id: nbqa-black
+        args: [--target-version=py39]
+      - id: nbqa-flake8
```

Comment:
All these hooks except flake automatically fix the files, so it should be easy to use these tools.